lib/purescript-cst/package.yaml

```diff
@@ -0,0 +1,77 @@
+name: purescript-cst
+version: '0.13.6'
+synopsis: PureScript Programming Language Concrete Syntax Tree
+description: The surface syntax of the PureScript Programming Language.
+category: Language
+author: Phil Freeman <paf31@cantab.net>
+maintainer: >
+  Gary Burgess <gary.burgess@gmail.com>,
+  Hardy Jones <jones3.hardy@gmail.com>,
+  Harry Garrood <harry@garrood.me>,
+  Christoph Hegemann <christoph.hegemann1337@gmail.com>,
+  Liam Goodacre <goodacre.liam@gmail.com>,
+  Nathan Faubion <nathan@n-son.com>
+copyright: (c) 2013-17 Phil Freeman, (c) 2014-19 Gary Burgess, (c) other contributors (see CONTRIBUTORS.md)
```

Comment:
Similar here about who should be in this list. Given that @natefaubion did the vast majority of the work, is this even accurate?

Reply:
I think so, at least in the case of anyone who worked on the parser and/or syntax before the CST work was done, because @natefaubion's work built off that.

Reply:
Makes sense. Thanks!